### DISCUSSION TITLE
Is there any plan to open some report of XTTS?

### DISCUSSION THREAD

**Discussion by seastar105** [coqui-ai/TTS#3190] [General Q&A] [answered]
Thanks for nice work and open-source its checkpoint!

It seems XTTS has tortoise-based architecture (mel vq-vae based GPT + gpt latent vocoder). we can know bit more as code is open, but paper-like stuff(e.g. technical report) of XTTS model would be great. is there any plan to open such stuff?

**Comment by erogol** [comment, marked as answer]
There is no paper but you can find some details on my substack https://erogol.substack.com/ or website https://erogol.com/

**Reply by seastar105** [reply]
Thank you.

**Reply by taorui-plus** [reply]
thank you for your contribution

**Comment by maituan** [comment]
hello, are there any plan for Xtts to support Vietnamese language

**Reply by erogol** [reply]
If there is data for us to use yes

**Comment by maituan** [comment]
Hi, what kind of text data do you need. I can support with Vietnamese

Tải xuống Outlook dành cho Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by erogol** [reply]
Can you find me on our discord channel and DM me?

**Comment by Edresson** [comment]
Hi @seastar105,

We finally were able to publish the XTTS paper available. It is available at [arXiv](https://arxiv.org/abs/2406.04904) and It will be presented at INTERSPEECH 2024.

**Reply by seastar105** [reply]
@Edresson It's really good to hear XTTS paper is out now! I read paper and have one question.

When training decoder, how did you generate GPT latent for training decoder? i think there're two way to generate it, one is input <text> <speech token> to GPT and use only <speech token> part latent, or only input <speech token> then use output. but not sure which one is proper.
